src/components/styles/Countries.module.css

```diff
@@ -0,0 +1,60 @@
+.linkList {
```

Comment:
![image](https://user-images.githubusercontent.com/79058364/217826455-b06fb502-f6f4-4064-8c34-ed92221dc9f1.png)

- [OPTIONAL] According to the project requirements  🙃  Let's make sure the Details page stick to the original design as much as possible 🎨 🖌️  

- You need to change your element flex Layout, and position. Please use flex for your text and image alignment
- You can also consider increasing the page width of your application
Your Implementation | Original Design
:---:|:---:
![image](https://user-images.githubusercontent.com/79058364/227736532-ff5f7361-0a69-41db-bd18-f27bf6fd2bd7.png)| ![image](https://user-images.githubusercontent.com/79058364/217828693-b49d57ec-1cb0-48d3-83e8-743823b60f19.png)